Add docs for `V1 Data Engine` and `Guaranteed Instance Manager CPU for V2 Data Engine` (#825)
longhorn/website: content/docs/1.6.0/best-practices.md

We recommend setting the CPU request for Longhorn instance manager pods.

### V1 Data Engine

> The `Guaranteed Instance Manager CPU` setting allows user to reserver percentage of the total allocatable CPU resources on each node for each instance manager pod when the V1 Data Engine is enabled. The default value is 12 percent.

Reviewer: ```suggestion
The `Guaranteed Instance Manager CPU` setting allows you to reserve a percentage of the total allocatable CPU resources on each node for each instance manager pod when the V1 Data Engine is enabled. The default value is 12.
```

Reviewer: When we mention default values, we are referring to the actual user input. In this case, the user is only required to type a number (without the % symbol).

You can also set a specific milli CPU value for instance manager pods on a particular node by updating the node's `Instance Manager CPU Request` field.

> **Note:** This field will overwrite the above setting for the specified node.

Refer to [Guaranteed Instance Manager CPU](../references/settings/#guaranteed-instance-manager-cpu) for more details.

### V2 Data Engine

> The `Guaranteed Instance Manager CPU for V2 Data Engine` setting allows user to reserve number of millicpus on each node for each instance manager pod when the V2 Data Engine is enabled. By default, the Storage Performance Development Kit (SPDK) target daemon within each instance manager pod uses 1 CPU core. Configuring a minimum CPU usage value is essential for maintaining engine and replica stability, especially during periods of high node workload. The default value is 1250 millicpus.

Reviewer: ```suggestion
The `Guaranteed Instance Manager CPU for V2 Data Engine` setting allows you to reserve a specific number of millicpus on each node for each instance manager pod when the V2 Data Engine is enabled. By default, the Storage Performance Development Kit (SPDK) target daemon within each instance manager pod uses 1 CPU core. Configuring a minimum CPU usage value is essential for maintaining engine and replica stability, especially during periods of high node workload. The default value is 1250.
```
